kratos/python/add_matrix_to_python.cpp

```diff
@@ -97,6 +97,7 @@ namespace Python
         matrix_binder.def(py::init<const DenseMatrix<double>::size_type, const DenseMatrix<double>::size_type, const DenseMatrix<double>::value_type >());
         matrix_binder.def("fill", [](DenseMatrix<double>& self, const typename DenseMatrix<double>::value_type value) { noalias(self) = DenseMatrix<double>(self.size1(),self.size2(),value); });
         matrix_binder.def("fill_identity", [](DenseMatrix<double>& self) { noalias(self) = IdentityMatrix(self.size1()); });
+        matrix_binder.def("transpose", [](DenseMatrix<double>& self) { return Matrix(trans(self)); });
```

Comment:
```suggestion
        matrix_binder.def("transpose", [](DenseMatrix<double>& self) { return trans(self); });
```

do you need the additional copy?

Reply:
otherwise i get an error

Reply:
can you post it?

Reply:
TypeError: Unable to convert function return value to a Python type! The signature was
        (self: Kratos.Matrix) -> boost::numeric::ublas::matrix_unary2<boost::numeric::ublas::matrix<double,boost::numeric::ublas::basic_row_major<unsigned __int64,__int64>,boost::numeric::ublas::unbounded_array<double,std::allocator<double> > >,boost::numeric::ublas::scalar_identity<double> >

Did you forget to `#include <pybind11/stl.h>`? Or <pybind11/complex.h>,
<pybind11/functional.h>, <pybind11/chrono.h>, etc. Some automatic
conversions are optional and require extra headers to be included
when compiling your pybind11 module.